src/neo/SmartContract/Native/Tokens/NeoToken.cs

```diff
@@ -64,7 +64,7 @@ private void DistributeGas(ApplicationEngine engine, UInt160 account, NeoAccount
 
             BigInteger gas = CalculateBonus(engine.Snapshot, state.VoteTo, state.Balance, state.BalanceHeight, engine.Snapshot.PersistingBlock.Index);
             state.BalanceHeight = engine.Snapshot.PersistingBlock.Index;
-            GAS.Mint(engine, account, gas);
+            GAS.Mint(engine, account, gas, true);
```

Comment:
Maybe we should set it `false`.

Reply:
Why?

Reply:
Because this part GAS is not from someone else, but from auto-claimed.